Source/RxComponents/LMCEquivalentCircuit.hh

```diff
@@ -0,0 +1,47 @@
+/*
+ * LMCEquivalentCircuit.hh
+ *
+ *  Created on: Nov 12, 2021
+ *      Author: jgaison
+ */
+
+#ifndef LMCEQUIVALENTCIRCUIT_HH_
+#define LMCEQUIVALENTCIRCUIT_HH_
+#include "param.hh"
+#include "LMCException.hh"
+#include "LMCConst.hh"
+#include "LMCSignal.hh"
+#include "LMCPatchAntenna.hh"
+#include "LMCSlotAntenna.hh"
+#include "LMCComplexFFT.hh"
+#include <vector>
+
+
+namespace locust
+{
+ /*!
+ @class LMCEquivalentCicuit
+ @author J. Gaison
+ @brief Base class to implement cavity parameterization
+ @details
+ Available configuration options:
```

Comment:
You can remove these lines about available configuration options; looks like they are all defined in LMCCavitySignalGenerator.

Reply:
These lines have been removed for your given reasons.